custom/icds_reports/tasks.py

```diff
@@ -236,7 +242,9 @@ def move_ucr_data_into_aggregation_tables(date=None, intervals=2):
             res_ccs.get()
             res_child.get()
 
-            res_awc = icds_aggregation_task.delay(date=calculation_date, func=_agg_awc_table)
+            res_awc = chain(icds_aggregation_task.si(date=calculation_date, func=_agg_awc_table),
```

Comment:
to rule out the condition where awc visited number exceed the awc launched. In case here if these two run independently on different tasks. if agg_awc fails. and agg_ls works then `awc visited number` can go higher than awc launched.